src/styles/global.css

```
@@ -1 +1,1073 @@
+        --text-secondary: #57534e;
+        --text-tertiary: #78716c;
+        --text-muted: #a8a29e;
+        --text-inverse: #fafaf9;
+
+        --border-subtle: rgba(0, 0, 0, 0.04);
+        --border-default: rgba(0, 0, 0, 0.08);
+        --border-strong: rgba(0, 0, 0, 0.15);
+
+        --font-display: 'Syne Variable', system-ui, sans-serif;
+        --font-heading: 'Syne Variable', system-ui, sans-serif;
+        --font-body: 'Roboto Slab Variable', Georgia, serif;
```

Stylelint (17.3.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)